pkg/proxy/command.go

```diff
@@ -0,0 +1,489 @@
+package proxy
```

Comment:
Nit: Is changing the filename worth losing git-blame? Git sees this is an add+delete rather than a move.